WordPress/Tests/Theme/NoAutoGenerateUnitTest.inc

```diff
@@ -0,0 +1,11 @@
+
+<p>Lubith</p>
+
```

Comment:
Are these "real life" examples or made up examples ? I would very much prefer to see real life examples here to demonstrate what is being sniffed for and how.

Also, let's add some examples to make sure we don't get false positives.